### PR TITLE
Creating a new dimension Region Name

### PR DESCRIPTION
**Pull request by SanjnaSrivatsaVMware**
@ljadvey to add more details in the supporting documents. I am pushing a  skeleton of that currently.

### CODE REVIEW

File: supporting_content/columns/regionname.md
Comment (udam-f2) [inline supporting_content/columns/regionname.md:3]: Lets not copy / paste the RegionId here. Instead in other places we've linked to that other field using a ../regionid.md type notation. Feel free to capture anything new here though.

**Comment by udam-f2** [comment]
> @ljadvey to add more details in the supporting documents. I am pushing a skeleton of that currently.

Added WIP to PR since this is not yet ready per the comment above.

**Comment by jpradocueva** [comment]
This PR has been moved to `5 days Review and Approval`. It needs the approval of at least three members and two maintainers to be considered as approved by the group.
@SanjnaSrivatsaVMware, @udam-f2, @ljadvey, could you please review the PR and approve it or provide your comments? Thanks

File: specification/columns/regionname.md
Comment (cnharris10) [inline specification/columns/regionname.md:5]: Similar as in https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/352, we need to say something like:

```
The Region Name column MUST be present in the billing data when the provider supports deploying resources or services within an *region*.
```

Udam has a few examples here: https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/352

**Review by cnharris10 — CHANGES_REQUESTED** [review]

File: specification/columns/regionname.md
Comment (cnharris10) [inline specification/columns/regionname.md:24]:
```suggestion
| FOCUS Essential | False            |
```

**Comment by jpradocueva** [comment]
@ljadvey do you have any comments on the latest proposal by Chris Harris?

File: supporting_content/columns/regionname.md
Comment (udam-f2) [inline supporting_content/columns/regionname.md:23]: can we get these filled out?

File: supporting_content/columns/regionname.md
Comment (udam-f2) [inline supporting_content/columns/regionname.md:53]: Rather than duplicating, let's reference the discussion in regionid.md

File: specification/columns/region.md
Comment (udam-f2) [inline specification/columns/region.md:1]: Let's rebase from working_draft or resolve conflicts here.

**Review by cnharris10 — APPROVED** [review]

**Review by mike-finopsorg — APPROVED** [review]
I like this addition to the spec for the reason that CSPs sometimes use their name vs Id in API responses and the like. So connecting data to the cloud bill can be challenging or require the practitioner to map Id to Name in order to associate data.

File: specification/columns/regionname.md
Comment (aqu-erp) [reply specification/columns/regionname.md:5]: What if a service is restricted to multiple regions? Example: multi-regional GCP resources where the data is replicated across regions defined in the instance configuration.

**Comment by jpradocueva** [comment]
This PR will be put forward for approval on Thursday, March 21.

File: specification/columns/.Rhistory
Comment (flanakin) [inline specification/columns/.Rhistory:1]: Is this supposed to be here?

File: specification/columns/regionname.md
Comment (flanakin) [inline specification/columns/regionname.md:5]:
```suggestion
RegionName MUST be present in the billing data when the provider supports deploying resources or services within an *region* and MUST be of type String. RegionName MUST NOT be null when a *resource* or *service* is operated in or managed from a distinct region by the Provider and MAY contain null values when a *resource* or *service* is not restricted to an isolated geographic area. RegionName MUST be consistent within the provider and MUST be the same value used to indicate the region when provisioning or purchasing the *resource* or *service*.
```

File: specification/columns/regionname.md
Comment (flanakin) [inline specification/columns/regionname.md:3]:
```suggestion
Region Name is a provider-assigned display name for an isolated geographic area where a [*resource*](#glossary:resource) is provisioned or a [*service*](#glossary:service) is provided. Region Name is commonly used for scenarios like analyzing cost and unit prices based on where *resources* are deployed.
```

**Review by flanakin — APPROVED** [review]

**Review by tobrien — APPROVED** [review]

File: specification/columns/.Rhistory
Comment (udam-f2) [reply specification/columns/.Rhistory:1]: the git graph for this is a wreck. Not sure where it came from but now fixed.

**Pull request merged**
